src/csharp/Microsoft.Spark/Sql/SparkSession.cs

```diff
@@ -223,5 +285,23 @@ public void Dispose()
         /// Stops the underlying SparkContext.
         /// </summary>
         public void Stop() => _jvmObject.Invoke("stop");
+
+        /// <summary>
+        /// Returns a single column schema of the given datatype.
+        /// </summary>
+        /// <param name="dataType">Datatype of the column</param>
+        /// <returns>Schema as StructType</returns>
+        private StructType SchemaWithSingleColumn(DataType dataType) =>
+            new StructType(new[] { new StructField("_1", dataType) });
+
+        /// <summary>
+        /// Converts rows of type T to <see cref="GenericRow"/> to return a 
+        /// <see cref="IEnumerable"/> of type <see cref="GenericRow"/>.
```

Comment:
This description is a bit confusing. Is there a way to make it more clear?  This method is transforming each element of `IEnumerable<T> input` into a single columned `GenericRow`.